kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/commands/application/ApplicationCommandRegistry.kt

```diff
@@ -225,9 +229,9 @@ public open class ApplicationCommandRegistry : KoinComponent {
 
                 if (existingCommand != null) {
                     when (commandObj) {
-                        is MessageCommand<*> -> messageCommands[existingCommand.id] = commandObj
-                        is SlashCommand<*, *> -> slashCommands[existingCommand.id] = commandObj
-                        is UserCommand<*> -> userCommands[existingCommand.id] = commandObj
+                        is MessageCommand<*> -> messageCommands.upsert(existingCommand.id, commandObj)
```

Comment:
You can implement this operator, if you didn't know. I think I'd recommend it haha